sage-ios/SAGE/SAGE/SignUpTableViewController.swift

```diff
@@ -74,7 +74,7 @@ class SignUpTableViewController: UITableViewController, UINavigationBarDelegate
     }
     
     func loadSchoolData() {
-        AnnouncementsOperations.loadSchools { (schoolDict) -> Void in
+        LoginHelper.loadSchools { (schoolDict) -> Void in
```

Comment:
LoginOperations doesnt have a loadSchools function so i just took the old one and put it in... is that ok